cron_main.py

```diff
@@ -63,5 +62,6 @@
                                ('/tasks/admin/clear_old_subs', AdminSubsClear),
                                ('/tasks/admin/enqueue/clear_old_webhooks', AdminWebhooksClearEnqueue),
                                ('/tasks/admin/clear_old_webhooks', AdminWebhooksClear),
+                               ('/tasks/admin/rebuild_district_teams/([0-9]+)', AdminCreateDistrictTeamsDo),
```

Comment:
I don't think we've been super consistent here. I guess since these are mostly triggered by admins, this is probably semantically correct.